website/content/en/docs/building-operators/helm/quickstart.md

```diff
@@ -3,321 +3,102 @@ title: Helm Based Operator Quickstart
 linkTitle: Quickstart
 weight: 2
 ---
+## Prerequisites
 
-This guide walks through an example of building a simple nginx-operator powered by [Helm][helm-official] using tools and libraries provided by the Operator SDK.
+- [helm][helm-install] version v2+.
+- [docker][docker_tool] version 17.03+.
+- [kubectl][kubectl_tool] version v1.11.3+.
+- [operator-sdk][operator_install] v1.0.0+
+- Access to a Kubernetes v1.11.3+ cluster.
 
-## Create a new project
-
-Use the CLI to create a new Helm-based nginx-operator project:
+## Creating a Project
 
+Create a directory, and then run the init command inside of it to generate a new project.
+ 
 ```sh
-$ mkdir nginx-operator
-$ cd nginx-operator
-$ operator-sdk init --plugins=helm --domain=com --group=example --version=v1alpha1 --kind=Nginx
+$ mkdir $GOPATH/src/nginx-operator
+$ cd $GOPATH/src/nginx-operator
```

Comment:
Helm operator has no requirement for Go. (In fact, even Go operators are not required to be created in GOPATH). Revert this.